rules/typescript.js

```diff
@@ -20,6 +20,17 @@ module.exports = {
      * definitions.
      */
     '@typescript-eslint/no-type-alias': 'off',
+    /*
+     * `ignoreRhs` option forces the choice between false positives and false
+     * negatives. Until this is fixed, enabling the option prevents false
```

Comment:
Should we add some TODO with an expiry date to monitor this?

Reply:
It will stop working once they release a new major version. This config option will be removed and invalid options are reported.